src/__tests__/app.test.ts

```diff
@@ -64,6 +64,7 @@ const aPagoPABasePath = "/pagopa/api/v1";
 const aMyPortalBasePath = "/myportal/api/v1";
 const aBPDBasePath = "/bpd/api/v1";
 const aCgnAPIBasePath = "/api/v1/cgn";
+const aGeoAPIBasePath = "/api/v1/geo";
```

Comment:
Should we be a little more specific here? Maybe `/api/v1/cgn/geo` or `/api/v1/cgn-geo` or `/api/v1/here`

Reply:
I'd rather maintain this naming since this set of APIS should be used by other App functionalities(i.e. services geolocation )